libcni/api_test.go

```diff
@@ -84,10 +84,13 @@ var _ = Describe("Invoking the plugin", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			Expect(result).To(Equal(&types.Result{
-				IP4: &types.IPConfig{
-					IP: net.IPNet{
-						IP:   net.ParseIP("10.1.2.3"),
-						Mask: net.IPv4Mask(255, 255, 255, 0),
+				IP: []*types.IPConfig{
+					{
+						Version: "4",
```

Comment:
No, it's just the Go struct version of the ReportResult in BeforeEach().  Since that doesn't report an Interface, we don't need one here.